packages/generators/templates/component/{{name}}.stories.{{mdx}}

```diff
@@ -0,0 +1,101 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { {{name}} } from ".";
+
+# {{ titleCase name }}
+
+```ts
+import { {{ name }} } from "@jobber/components/{{ name }}";
+```
+
+<Canvas>
+  <Story name="{{ name }}" args={ { "text": "Bob"} }>
```

Comment:
If someone has the linter on this should remove the bracket spaces anyway but...
```suggestion
  <Story name="{{ name }}" args={{ "text": "Bob"}}>
```